src/webui/btjson.cpp

```diff
@@ -143,6 +144,17 @@ static const char KEY_TRANSFER_UPRATELIMIT[] = "up_rate_limit";
 static const char KEY_TRANSFER_DHT_NODES[] = "dht_nodes";
 static const char KEY_TRANSFER_CONNECTION_STATUS[] = "connection_status";
 
+static const char KEY_FULL_UPDATE[] = "full_update";
+static const char KEY_RESPONSE_ID[] = "rid";
+static const char KEY_SUFFIX_REMOVED[] = "_removed";
+
+QVariantMap getTranserInfoMap();
+QVariantMap toMap(const QTorrentHandle& h);
+void processMap(QVariantMap prevData, QVariantMap data, QVariantMap &syncData);
+void processHash(QVariantHash prevData, QVariantHash data, QVariantHash &syncData, QVariantList &removedItems);
+void processList(QVariantList prevData, QVariantList data, QVariantList &syncData, QVariantList &removedItems);
```

Comment:
>So wouldn't it be clearer to pass them by const reference(```const &```)?

Copy of ```prevData``` data is edited into the functions. Qt using Implicit Sharing (see [here](http://qt-project.org/doc/qt-4.8/implicit-sharing.html)). I think that such the functions declaration is also appropriate.